dataprofiler/tests/profilers/abstract_test_options.py

```diff
@@ -1,6 +1,5 @@
 from dataprofiler.profilers.profiler_options import BaseOption
 
-
 class AbstractTestOptions():
```

Comment:
There should be 2 lines between imports and code

Reply:
Oh! I did not know that, let me fix all of those.